packages/web/src/apollo/index.tsx

```diff
@@ -188,12 +195,26 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
     httpLink = new HttpLink({ uri, fetch: crossFetch, ...httpLinkConfig })
   }
 
-  // The order here is important. The last link *must* be a terminating link like HttpLink.
+  // Our terminating link needs to be smart enough to handle subscriptions, and if the GraphQL query
+  // is subscription it needs to use the SSELink (server sent events link).
+  const terminatingLink = new RetryLink().split(
```

Comment:
Using a `RetryLink` because of the known gotcha you mentioned?

Reply:
Retry link not needed. There is retry support within SSELink by default.

Reply:
The terminating link simply uses salt to determine if the operation is a subscription or not.